### PR TITLE
RM-269 add v3.0.4 release notes

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

File: docs/release_notes.rst [hunk 1]
```diff
@@ -1,6 +1,87 @@
 Release Notes
 *************
 
+v3.0.4 released 2017-08-01
```
Comment (hrvojem) [inline docs/release_notes.rst]: 2017-08-02

File: Changelog [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Changelog for Percona Toolkit
 
+v3.0.4 released 2017-08-01
```
Comment (hrvojem) [inline Changelog]: 2017-08-02

File: docs/release_notes.rst (hunk 1 above)
Comment (delgod) [inline docs/release_notes.rst]: fixed

File: Changelog (hunk 2 above)
Comment (delgod) [inline Changelog]: fixed

**Comment by hrvojem** [comment]
LGTM

**Pull request merged**